### ISSUE TITLE
Add weather

### ISSUE THREAD

**Issue by DougManuel** [Big-Life-Lab/PHES-ODM#204]
Add EU implementation of weather as a measure.

**Comment by DougManuel** [comment]
@sorinsion could you tell us what the categories or allowable responses for this measure are? Is there a description of the measure?

**Comment by sorinsion** [comment]
Hmm, seems to be free text. This is what I picked up:

*Weather*

sunny

31°C

dry weather, 32°C

sunny, dry

warm 30°C

dry day

rainy, 13°C

cloudy, some rain in the night 14°C

dry 22°C from 22.00 thunderstorm

dry

22°C, Humidy 69,4%, Mostly clear

morning: sunny weather, midday: cloudy, evening: rainy

rainy

sunny, windy

clear dry, 12-17°C, wind NW average 10 m/s gusts 22 m/s

sun

-

hot, rainy, cloudy, dry, windy T=20°C

26,8/13,5°C sunny

soleado

mostly sunny, 9°C-20°C

partly cloudy, mostly sunny, no rain


Sorin Sion
Tel: (+4)0747.262.661

**Comment by mathew-thomson** [comment]
While still maintaining measures for envTemp, envSnwD, wwFlow, wwTemp, wwTSS, and wwCond (as mentioned in issue #124), we've now also added a weather measure that captures more qualitative weather attributes (ie: Sunny, Rainy, Cloudy, Foggy, snowy) as mentioned in Sorin's comment above. The specific temperature measures were already possible to capture. Measure for wind speed and humidity were also added.
